src/celestia/celx_phase.cpp

```diff
@@ -98,7 +73,7 @@ static int phase_timespan(lua_State* l)
 
     celx.checkArgs(1, 1, "No arguments allowed for to phase:timespan");
 
-    const TimelinePhase* phase = this_phase(l);
+    auto& phase = *this_phase(l);
```

Comment:
reference to pointer == pointer to pointer

Reply:
Actually this is a reference to shared_ptr.